src/main/kotlin/com/terraformation/backend/report/db/ReportStore.kt

```diff
@@ -380,6 +384,39 @@ class ReportStore(
     }
   }
 
+  /** Save the seed bank buildStartDate, buildCompletedDate, and operationStartDate */
+  private fun saveSeedBankInfo(body: ReportBodyModel) {
+    val reportBody = body.toLatestVersion()
+    reportBody.seedBanks
+        .filter { it.selected }
+        .forEach {
+          dslContext
+              .update(FACILITIES)
+              .set(FACILITIES.BUILD_STARTED_DATE, it.buildStartedDate)
+              .set(FACILITIES.BUILD_COMPLETED_DATE, it.buildCompletedDate)
+              .set(FACILITIES.OPERATION_STARTED_DATE, it.operationStartedDate)
+              .where(FACILITIES.ID.eq(it.id))
+              .execute()
+        }
+  }
+
+  /** Save the nursery buildStartDate, buildCompletedDate, operationStartDate, and capacity */
+  private fun saveNurseryInfo(body: ReportBodyModel) {
+    val reportBody = body.toLatestVersion()
+    reportBody.nurseries
+        .filter { it.selected }
+        .forEach {
+          dslContext
+              .update(FACILITIES)
+              .set(FACILITIES.BUILD_STARTED_DATE, it.buildStartedDate)
```

Comment:
Pardon my ignorance.. if there's a nursery in the report body, does that imply the dates have changed? Or can we check the values to optimize/skip-saving, etc.

Reply:
Yeah, I wondered about this. Seems like I should query for the existing facility and do a comparison check against the existing values to see if anything changed before updating